### DISCUSSION TITLE
Graph API obsolete

### DISCUSSION THREAD

**Discussion by srijken** [ravendb/ravendb#14589] [General]
I noticed the obsoletion of the Graph API today, causing a few questions.

What's the reason for the obsoletion? Too hard to scale / index this right? Not used enough?

Will there be a alternative to this?

Thanks!

**Comment by ayende** [comment]
There wasn't enough usage in the field to justify keeping it.
We are in the process of updating the search engine we use, and graph queries made it much more complex.

The same scenarios are still in place, since you can do graph walks via `load` calls in the query, though.

What kind of queries are you running?
